cmd/pomerium/main.go

```diff
@@ -9,7 +9,7 @@ import (
 
 	"github.com/fsnotify/fsnotify"
 	"github.com/gorilla/mux"
-	"github.com/spf13/viper"
+	"github.com/pomerium/viper"
```

Comment:
Not that viper is much of a moving target, but can we avoid forking it?

Reply:
Probably not, unless we are okay with the potential race conditions. I can try to make these changes and request changes upstream but I'm not sure of the appetite.

Some context.
- https://github.com/spf13/viper/issues/19
- https://github.com/spf13/viper/issues/174
- https://github.com/googleforgames/open-match/issues/304
- https://github.com/spf13/viper/issues/629
- https://github.com/spf13/viper/issues/378
- https://github.com/spf13/viper/issues/353

Reply:
Probably not, unless we are okay with the potential race conditions. I can try to make these changes and request changes upstream but I'm not sure of the appetite.

Some context.
- https://github.com/spf13/viper/issues/19
- https://github.com/spf13/viper/issues/174
- https://github.com/googleforgames/open-match/issues/304
- https://github.com/spf13/viper/issues/629
- https://github.com/spf13/viper/issues/378
- https://github.com/spf13/viper/issues/353

Reply:
I've undone these changes both here and in #328 based on our conversation regarding moving viper into the options struct to be instance based.